game/scripts/vscripts/components/musicplayer/musicplayercontrol.lua

```diff
@@ -0,0 +1,68 @@
+--[[
+  Author:
+    Relacibo
+]]
+
+if MusicPlayerControl == nil then
+  DebugPrint ( '[musicplayer/musicplayercontrol] Creating new MusicPlayerControl object' )
```

Comment:
while you're in here you can remove the stuff the "`[musicplayer/musicplayercontrol]`" since the new debugging system automatically strips it out and replaces it with the filename and line number